src/config.rs

```diff
@@ -119,16 +79,14 @@ pub(crate) fn parse_config() -> Result<Config> {
     let config: Config = toml::from_str(&std::fs::read_to_string(config_path)?)?;
 
     // validity checks
-    if config.farmer.plot_size < "1GB".parse::<ByteSize>().expect("hardcoded value is true") {
-        return Err(Report::msg("size should be bigger than 1GB!"));
+    if config.farmer.plot_size < ByteSize::gib(1) {
+        return Err(eyre!("size should be bigger than 1Gib!"));
```

Comment:
I appreciate the specific-ness of 1Gib here but wondering if that might be confusing to some. 1GB from a broader sense is more common for folks to see (yes the size is different but close enough for the audience IMO).

Also, a bit confused, but should this be `100GB`?

https://github.com/subspace/subspace-cli/blob/c8f23d1143152dabdbdef793c8c930d17edad1cb/src/commands/init.rs#L12

Reply:
The thing isб I wanted to accept both 1Gib and 1GB, but 1Gib is less than 1GB. As for 100GB it would be better to encourage ppl to pledge space in GB as it would have some nice number of sectors there.

Reply:
I agree with @achiurizo here, both for the users and for us, sticking with a single terminology should be better I guess. The code will be less prone to errors caused by developer's confusion